app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -164,7 +164,10 @@ case class Choice(
   orientation: Orientation,
   selections: List[Int],
   hints: Option[NonEmptyList[SmartString]],
-  optionHelpText: Option[NonEmptyList[SmartString]]
+  optionHelpText: Option[NonEmptyList[SmartString]],
+  dividerPositon: Option[Int],
+  dividerText: String = "or",
```

Comment:
This cannot be `String` since `String` cannot be localised. Use `LocalisedString` instead.